tests/epics/adcore/test_drivers.py

```diff
@@ -93,3 +93,48 @@ async def wait_then_fail():
         ValueError, match="Final detector state Disconnected not in valid end states:"
     ):
         await acquiring
+
+
+@patch("ophyd_async.core._detector.DEFAULT_TIMEOUT", 0.2)
```

Comment:
Please could we add `timeout=DEFAULT_TIMEOUT` as an argument to `start_acquiring_driver_and_ensure_status` with docstring `timeout used for waiting for the driver to start acquiring, and for waiting for the detector to detector to be in a good state after it stops acquiring` then use it here instead of patching.

Reply:
Added in https://github.com/bluesky/ophyd-async/pull/889/commits/8111d62fdbea4c5bb5be492e82e6dd5dccde63ab. 

The full timeout is used for the first call to set the detector acquire. Then any remaining timeout is used to wait for the detector to return to a good status.